shardingsphere-test/shardingsphere-parser-test/src/main/resources/sql/supported/ral/common.xml

```diff
@@ -52,6 +52,6 @@
     <distsql-case id="prepare-distsql" value="PREPARE DISTSQL" />
     <distsql-case id="apply-distsql" value="APPLY DISTSQL" />
     <distsql-case id="discard-distsql" value="DISCARD DISTSQL" />
-    <distsql-case id="alter-instance-xa-recovery-id" value="ALTER INSTANCE 1.2.3.4@3307 SET XA_RECOVERY_ID=ID_TEST" />
+    <distsql-case id="alter-instance-xa-recovery-nodes" value="ALTER INSTANCE 1.2.3.4@3307 SET XA_RECOVERY_NODES=1.2.3.4@3308" />
```

Comment:
It maybe better to add a case like
```
ALTER INSTANCE 1.2.3.4@3307 SET XA_RECOVERY_NODES=1.2.3.4@3308,1.2.3.4@3307;
```

Reply:
it was provided to openForce https://github.com/apache/shardingsphere/issues/16357